### DISCUSSION TITLE
Data importer wrong date

### DISCUSSION THREAD

**Discussion by andresatierf** [firefly-iii/firefly-iii#9042] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Similar to #1329. Used data importer, transactions were created with date the day before at 23h00 instead of the right day at midnight.

### Debug information

Debug information generated at 2024-07-12 01:30:03 for Firefly III version **v6.1.18**.

<table><thead><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Firefly III</td><td>6.1.18 / v2.1.0 / 24 (exp. 24)</td></tr><tr><td>PHP version</td><td>8.3.8 (64bits) / apache2handler / Linux x86_64</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Error reporting</td><td>Display: Off, reporting: ALL errors</td></tr><tr><td>Max upload</td><td>67108864 (64 MB)</td></tr><tr><td>Database drivers</td><td>mysql, pgsql, <strong>*sqlite*</strong>, </td></tr><tr><td>Docker build</td><td><span>#</span>1104, base <span>#</span>85</td></tr></tbody></table><table><thead><tr><th colspan="2">Firefly III information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Timezone</td><td>Europe/Lisbon + Europe/Lisbon</td></tr><tr><td>App environment</td><td>production, debug: false</td></tr><tr><td>Layout</td><td>v1</td></tr><tr><td>Logging</td><td>notice, stack / (empty)</td></tr><tr><td>Cache driver</td><td>file</td></tr><tr><td>Default language and locale</td><td>en_US + equal</td></tr><tr><td>Trusted proxies</td><td>**</td></tr><tr><td>Login provider & user guard</td><td>eloquent / web</td></tr><tr><td>Login headers</td><td>N/A + N/A</td></tr><tr><td>Stateful domains</td><td></td></tr><tr><td>Last cron job</td><td>never (never)</td></tr><tr><td>Mailer</td><td>log</td></tr></tbody></table><table><thead><tr><th colspan="2">User-specific information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>User</td><td><span>#</span>1 of 1</td></tr><tr><td>User flags</td><td></td></tr><tr><td>Session start</td><td>2024-07-01 00:00:00</td></tr><tr><td>Session end</td><td>2024-07-31 23:59:59</td></tr><tr><td>View range</td><td>1M</td></tr><tr><td>User language</td><td>en_US</td></tr><tr><td>User locale</td><td>en_US</td></tr><tr><td>Locale(s) supported</td><td>en_US.utf8: :white_check_mark:<br>en_US.UTF-8: :white_check_mark:<br></td></tr><tr><td>User agent</td><td>Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:128.0) Gecko/20100101 Firefox/128.0</td></tr></tbody></table>

### Expected behaviour

I expected the dates to be correct.

### Additional info

I assume there is a mismatch regarding the daylight savings

**Comment by JC5** [comment, marked as answer]
I guess the data importer has a different time zone from Firefly III, that's usually the cause.

**Reply by andresatierf** [reply]
you're right, sorry to bother
